content/02.main-text.md

```diff
@@ -1,3 +1,21 @@
 ## Main text {.page_break_before}
 
-*Work in progress*
+### Introduction
+
+Open research – which includes sharing code, data, and manuscripts – benefits the researchers who engage in open practices [@doi:10.7554/eLife.16800], their scientific peers, and the public.
+`TODO: more references needed`
+Here we advocate for the benefits of writing review articles openly, specifically planning, organizing, writing, and editing this type of manuscript collaboratively in an unrestricted public forum.
+Reviews presenting the state of the art in a scientific field are often prepared by a single research group or a small team of colleagues.
+In contrast, broadly opening the writing process to anyone engaged in the topic can help maximize the review's value by ensuring that it represents diverse opinions and does not neglect relevant research.
+Review authors can engage with the authors of original research to clarify their methods and results and present them accurately.
```

Comment:
How about using greenelab/deep-review/issues/213 instead to show that it wasn't just me, a meta-review author, having these discussions?  I'm okay referencing a GitHub issue, but it would be nice to have an archival version of it.  That conversation could be edited by either party.